blog-service/2023-05-11-traces.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Enhanced Trace Query Visualizations (Traces)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - traces
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We've enhanced the Trace Query Visualizations screen by adding five new aggregation charts alongside the existing Trace duration breakdown chart. These charts provide more insights into the behavior and performance of your application. Aggregating traces allows you to quickly identify anomalies and unexpected behaviors, resulting in a shorter Time-to-Resolution (TTR) and a higher Return on Investment (ROI).
+
+The new aggregate charts are:
+* Trace duration
+* Error count
+* Span count
+* Trace duration as timeseries
+* Trace duration as histogram
+
+To access the new charts, simply navigate to the Trace Query Visualizations screen and select the **Trace duration** dropdown menu. From there, you can choose from the **duration/errors/spans** (per trace) and **timeseries/histogram** options. With these choices, you'll now have a total of six charts to help you better understand the profile of your traces.
+
+NOTE: We recommend filtering your trace data to represent transactions for the best results.
+
+[Learn More](https://help.sumologic.com/docs/apm/traces/view-and-investigate-traces/#trace-duration-breakdown-chart)
```

Comment:
While you were out on leave, we switched from absolute to rel. links in release notes.

```suggestion
[Learn More](/docs/apm/traces/view-and-investigate-traces/#trace-duration-breakdown-chart)
```

Reply:
@kimsauce Thank you. That makes sense. Style should be the same for all the docs.